dev/package-examples/iptables-1.0.4/docs/README.md

```diff
@@ -22,40 +17,31 @@ a structure suitable for visualizing in Kibana.
 
 include::../include/running-modules.asciidoc[]
 
-[float]
-=== Example dashboard
+## Example dashboard
 
 This module comes with sample dashboards showing geolocation and network
 protocols used. One for all iptables logs:
 
-[role="screenshot"]
-image::./images/kibana-iptables.png[]
 
 and one specific for Ubiquiti Firewall logs:
 
-[role="screenshot"]
-image::./images/kibana-iptables-ubiquiti.png[]
-
-include::../include/configuring-intro.asciidoc[]
+![Image in text](../img/kibana-iptables-ubiquiti.png "Image in text")
```

Comment:
I believe so. It should just work because Markdown supports it. As long as we don't need to do any extra processing, I'm for it. It's no riskier than the other user generated/supplied content we're hosting.

Reply:
Ok, lets see what happens :-) In general we should encourage package creators to use the screenshots feature.